src/datasource.tsx

```diff
@@ -319,7 +319,7 @@ export class PrometheusDatasource
         continue;
       }
 
-      target.requestId = options.panelId + target.refId;
+      target.requestId = options.requestId
```

Comment:
actually, at this line 
https://github.com/grafana/grafana/blob/main/public/app/features/inspector/QueryInspector.tsx#L56
grafana will check requestId is same or not. if same, it will update inspector response. so we should use original reuqestId to return.